### DISCUSSION TITLE
The year winter came late

### DISCUSSION THREAD

**Discussion by ghost** [community/community#20296] [Other Feature Feedback, Questions, & Ideas]
What kind of book is it and what is it about?

**Comment by Nikka13** [comment]
Game of Thrones

**Comment by y9su** [comment]
It's a poem I believe.

**Reply by ghost** [reply]
It started as poem but not a poem now.

**Comment by radar-blip** [comment]
Paperback. Looks like a childrens book.
Summary:
The Year Winter Came Late is a story about a town that had its winter season stolen by a mean old snowman named Mr. Sneezy Snew. Just when all the people thought there was no hope in getting their winter back a little girl calls out from the middle of the crowd saying, "My name is Lisa, and I know what to do, I know who has your winter he is cranky old snowman whose name is Mr. Sneezy Snew." Read how Lisa and three friends, Mr. Red Robin, Mr. Gray Squirrel, and Mr. Brown Bear outsmart Sneezy Snew in a game of riddles to get him to bring winter back to Snowtown.

P.S. I think you already know this. 👍🏼

**Comment by ghost** [comment]
It is a children's book

**Reply by radar-blip** [reply]
Correct!

**Reply by ghost** [reply]
I wrote the book. there will be three more books in the set. Can you guess the titles of the other three books?

**Reply by radar-blip** [reply]
Mr Sneezy goes to Florida
